test/test_expr.jl

```diff
@@ -268,8 +268,8 @@ function test_extension_MA_add_mul(
     @variable(model, x)
     @variable(model, y)
     # MA.add_mul!!(ex::Number, c::Number, x::GenericAffExpr)
-    aff = MA.add_mul!!(1.0, 2.0, JuMP.GenericAffExpr(1.0, :a => 1.0))
-    @test JuMP.isequal_canonical(aff, JuMP.GenericAffExpr(3.0, :a => 2.0))
+    aff = MA.add_mul!!(1.0, 2.0, JuMP.GenericAffExpr(1.0, x => 1.0))
```

Comment:
@odow Was that intentional ? I could also define `value_type(::Symbol)` to make it work

Reply:
I didn't change the content of the tests. So I assumed it was intentional. I didn't track the original introduction.